### PR TITLE
Fixes to BBC Weather icons and text

### PR DESCRIPTION
**Pull request by georgejkaye**
Made it so that the BBC weather (for bbc.co.uk/weather) icons don't have white backgrounds any more, and made the text in the wind speed and pollen, uv, pollution icons dark text so it's readable.

(There's also a tiny fix that makes the results in Detexify inverted)

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -375,6 +375,23 @@ INVERT
...
+.wr-icon-weather-type__svg-background, 
+.wr-icon-rain__svg-background,
+.wr-icon-wind-direction__svg-background,
+.wr-icon-gel__svg-background {
+    opacity: 0%
+}
+
+.wr-value--windspeed,
+.wr-forecast .wr-c-environmental-data__icon-text  {
+    color: #181a1b
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:390]: Please use `color: ${color to be inverted}`

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -806,6 +823,13 @@ INVERT
 
 ================================
 
+detexify.kirelabs.org
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:826]: Rule 5 https://github.com/darkreader/darkreader/wiki/Merging-Pull-Requests

**Comment by georgejkaye** [comment]
Hope that's all the fixes done, I should read the rules more carefully next time!

**Comment by georgejkaye** [comment]
Hopefully that's everything fixed now, I should read the rules more carefully next time!

**Comment by Gusted** [comment]
Redirecting to .com

**Comment by georgejkaye** [comment]
It's the opposite for me - bbc.com redirects to bbc.co.uk! Probably since I'm in the UK (https://www.bbc.co.uk/contact/questions/behind-the-scenes/website-domain-change). I'm not sure how different the two sites are - and changing the site to bbc.com for me does not apply the changes for me.

Here is two screenshots of the before and after for the site I see - https://imgur.com/a/gp1rVcv

**Comment by Gusted** [comment]
Interesting

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -375,6 +375,26 @@ INVERT
 
 ================================
 
+bbc.co.uk/weather
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:379]: Just add bbc.com/weather so

```
bbc.co.uk/weather
bbc.com/weather

CSS
```

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -375,6 +375,26 @@ INVERT
...
+.wr-c-environmental-data__icon-text {
+    color: ${#dcd9d4} !important;
+}
+
+================================
+
 bbc.com/news
 bbc.com/sport
 bbc.com/weather
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:400]: Make sure to remove this line

and merge the 2 invert what this section contain to above as well so it stay the same.

**Pull request merged**